Examples/Retired iOS Demo/Retired iOS Demo/AppDelegate.swift

```diff
@@ -50,4 +53,8 @@ extension Message {
   private func goToAppStore(action: UIAlertAction) {
     UIApplication.sharedApplication().openURL(iTunesURL)
   }
+
+  private func doItAgainLater(action: UIAlertAction) {
+    Retired.suppressUntil(NSDate(timeIntervalSinceNow: intervalBetweenRequests))
+  }
```

Comment:
I thought about that. To be clear, you're suggesting adding an interval to the configure call (with a default) and just setting it every time a successful fetch is made?